### PR TITLE
installation guide update

### PR DESCRIPTION
**Pull request by derisen**

### CODE REVIEW

File: source/index.rst [hunk 1]
```diff
@@ -13,5 +13,4 @@ Welcome to DHIL Documentation Guide's documentation!
    intro
    tools
    markup
-   contributors
-   todo
```
Comment (ubermichael) [inline source/index.rst]: The To Do file is important, and should not be removed.

File: source/todo.rst [hunk 2]
```diff
@@ -1,6 +0,0 @@
-.. _todo-label:
```
Comment (ubermichael) [inline source/todo.rst]: This file is important and should not be removed.

**Comment by derisen** [comment]
I've made some similar mistakes as in ceww-docs. Will correct and recommit!

File: source/tools.rst [hunk 3]
```diff
@@ -101,6 +101,12 @@ Once python and pip are set up, you can install Sphinx by using the following co
 
   $ pip install Sphinx
 
+Run the following command to setup Sphinx's environmental variables:
+
+.. code-block:: console
+
+  $ sphinx-quickstart
```
Comment (ubermichael) [inline source/tools.rst]: This command is only used to start a new documentation project. I don't think this part of the instructions are for starting a new project, just getting an existing one up and running.

**Review by ubermichael — CHANGES_REQUESTED** [review]
This mostly looks pretty good, but there are some changes to be made.

**Comment by derisen** [comment]
Got it!

File: source/index.rst (hunk 1 above)
Comment (derisen) [reply source/index.rst]: restored.

File: source/todo.rst (hunk 2 above)
Comment (derisen) [reply source/todo.rst]: restored.

File: source/tools.rst (hunk 3 above)
Comment (derisen) [reply source/tools.rst]: removed.

**Comment by ubermichael** [comment]
This looks good to me Doğan. You should add yourself as a contributor to `contributors.rst` following Erik's example, unless you'd prefer not to.

**Pull request merged**